questionpy/_ui.py

```diff
@@ -6,11 +6,12 @@
 import jinja2
 
 from questionpy_common.environment import Package, PackageNamespaceAndShortName, get_qpy_environment
-from questionpy_sdk.constants import TEMPLATES_DIR
 
 if TYPE_CHECKING:
     from questionpy import Attempt, Question
 
+TEMPLATES_DIR = "templates"
```

Comment:
Dieses `TEMPLATES_DIR` und das in der `constants.py` wird jeweils nur an einer einzigen Stelle verwendet. Damit könnten wir beide auch direkt inlinen.